qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -924,14 +1234,25 @@ def parameters_table(
 
             keys.add(key)
 
-        for key in keys:
-            for value in self._params[key]:
-                value_dict = dataclasses.asdict(value)
-                value_dict["qubits"] = key.qubits
-                value_dict["parameter"] = key.parameter
-                value_dict["schedule"] = key.schedule
-                value_dict["date_time"] = value_dict["date_time"].strftime("%Y-%m-%d %H:%M:%S.%f%z")
-                data.append(value_dict)
+        # The following dictionary is used to keep track of the most recent parameter values.
+        data = []
+        if most_recent_only:
+            most_recent = dict()
+
+            for key in keys:
+                for value in self._params[key]:
+                    if key not in most_recent:
+                        most_recent[key] = value
+                    elif value.date_time > most_recent[key].date_time:
+                        most_recent[key] = value
```

Comment:
```suggestion
                most_recent[key] = sorted(self._params[key].values(), key=lambda x: x.date_time)[0]
```
probably this is what you wanted to write?

Reply:
Using `max()` instead of `sorted()` (without the `[-1]`) would save the work of sorting the lists.

Reply:
Taking both of your suggestions into account. This is what comes out: e98ee65